client/src/lib/components/FeedbackCarousel/FeedbackCrousel.tsx

```diff
@@ -0,0 +1,99 @@
+import dayjs from 'dayjs';
+import React, {useCallback} from 'react';
+import {FlatList} from 'react-native-gesture-handler';
+import {ListRenderItem} from 'react-native';
+import styled from 'styled-components/native';
+
+import {COLORS} from '../../../../../shared/src/constants/colors';
+import {Feedback} from '../../../../../shared/src/types/Feedback';
+import SETTINGS from '../../constants/settings';
+import {SPACINGS} from '../../constants/spacings';
+import {Spacer8} from '../Spacers/Spacer';
+import Tag from '../Tag/Tag';
+import {
+  ThumbsUpWithoutPadding,
+  ThumbsDownWithoutPadding,
+} from '../Thumbs/Thumbs';
+import {Body16} from '../Typography/Body/Body';
+
+const CARD_WIDTH = 216;
+
+const List = styled(FlatList)({
+  flexGrow: 0,
+  width: '100%',
+}) as unknown as FlatList;
+
+const CardWrapper = styled.View<{isLast: boolean}>(({isLast}) => ({
+  paddingLeft: SPACINGS.SIXTEEN,
+  paddingRight: isLast ? SPACINGS.SIXTEEN : undefined,
+}));
+
+const Card = styled.View({
+  ...SETTINGS.BOXSHADOW_SMALL,
+  backgroundColor: COLORS.WHITE,
+  borderRadius: SETTINGS.BORDER_RADIUS.CARDS,
+  padding: SPACINGS.SIXTEEN,
+  width: CARD_WIDTH,
+  minHeight: 120,
+  maxHeight: 216,
+  marginBottom: SPACINGS.SIXTEEN,
+});
+
+const Row = styled.View({
+  flexDirection: 'row',
+  alignItems: 'center',
+});
+
+const DateTag = styled(Tag)({
+  marginTop: 0,
+  backgroundColor: COLORS.PURE_WHITE,
+});
+
+const ThumbWrapper = styled.View({
+  width: SPACINGS.TWENTYFOUR,
+  height: SPACINGS.TWENTYFOUR,
+});
+
+const FeedbackCarousel: React.FC<{feedbackItems: Feedback[]}> = ({
+  feedbackItems,
+}) => {
+  const keyExtractor = useCallback((item: Feedback) => item.id, []);
+
+  const renderItem = useCallback<ListRenderItem<Feedback>>(
+    ({item, index}) => (
+      <CardWrapper isLast={index === feedbackItems.length - 1}>
+        <Card>
+          <Row>
+            <ThumbWrapper>
+              {item.answer ? (
+                <ThumbsUpWithoutPadding />
+              ) : (
+                <ThumbsDownWithoutPadding />
+              )}
+            </ThumbWrapper>
+            <Spacer8 />
+            <DateTag>{dayjs(item.createdAt).format('d MMM')}</DateTag>
```

Comment:
I don't think this is locale safe. Check these :) https://day.js.org/docs/en/display/format#localized-formats

Reply:
Seems like none of those match the design format 😕

Reply:
Shall we update all of these to use locale formats?
<img width="506" alt="Screenshot 2023-09-07 at 15 55 19" src="https://github.com/29ki/29k/assets/7523828/20bf6223-6c1b-46d0-afba-33308534e7b1">

Reply:
Never mind I got it wrong... They do get localised but does not change the order of things based on locale. So this `format('ddd, D MMM HH:mm')` actually gets localised correctly in this screenshot ("tor")
![Uploading image.png…]()

Reply:
Never mind I got it wrong... They do get localised but does not change the order of things based on locale. So this `format('ddd, D MMM HH:mm')` actually gets localised correctly in this screenshot ("tor")
<img width="524" alt="image" src="https://github.com/29ki/29k/assets/474066/bbbab2be-929f-4347-baea-836862001928">

Reply:
aha, cool!